env/gpu/py3-master.yml

```diff
@@ -33,7 +33,7 @@ dependencies:
     - flaky==3.6.1
     - flake8==3.7.9
     - mock<3
-    - https://repo.mxnet.io/dist/python/cu100/mxnet_cu100-1.6.0-py2.py3-none-manylinux1_x86_64.whl
+    - https://repo.mxnet.io/dist/python/cu100/mxnet_cu100-1.7.0b20200618-py2.py3-none-manylinux2014_x86_64.whl
```

Comment:
- mxnet-cu100mkl

Reply:
Thank you very much for the suggestion.
However, this PR intends to test MXNet 1.7 and the BERT-GPU improvements available in that version.
I cannot find a MXNet 1.7 wheel on https://dist.mxnet.io/python that includes both MKL and CUDA.
I was trying to fix the issue with this PR https://github.com/apache/incubator-mxnet/pull/18608, but got blocked as well.
Please, any help is very welcome.
Thank you.

Reply:
Ah, that seems a tough problem. 
Maybe currently we have to stash this pull request and wait for some other releases of the dependencies to be ready.